### PR TITLE
CompareVersion Solution Added

### PR DESCRIPTION
**Pull request by Deeeeksha**
Hi @qiyuangong  I have added CompareVersion Solution . KIndly review the changes

### CODE REVIEW

**Comment by qiyuangong** [comment]
Hi @Deeeeksha 
Thank you for contribution! Nice solution indeed.
Pls change file name into `165_Compare_Version_Numbers.py`, and apply code style/format with IDE's help.

File: python/165_CompareVersion.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Solution:
+    def CompareVersion(self, version1: str, version2: str) -> int:
```
Comment (qiyuangong) [inline python/165_CompareVersion.py]: Change to `compareVersion` to avoid submission error

File: python/165_CompareVersion.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            return(0)
+        
+        a=len(l1)
+        b=len(l2)
+        
+        if a>b:
+            for i in range(a-b):
+                l2.append("0")
+        
+        if b>a:
```
Comment (qiyuangong) [inline python/165_CompareVersion.py]: this if can be change to else.

**Comment by Deeeeksha** [comment]
Thankyou @qiyuangong  I will make the changes as requested.

**Comment by Deeeeksha** [comment]
Hello @qiyuangong. Please review the changes requested as I have updated the code.

**Comment by Deeeeksha** [comment]
Hello @qiyuangong , Any updates?

**Review by qiyuangong — APPROVED** [review]
LGTM

**Comment by Deeeeksha** [comment]
Thankyou  @qiyuangong .

**Comment by qiyuangong** [comment]
> Thankyou @qiyuangong .

You are welcome :)

Thank you for your contribution!

**Pull request merged**